### DISCUSSION TITLE
Organization: Extract KSTO from Streaming tab

### DISCUSSION THREAD

**Discussion by hawkrives** [StoDevX/AAO-React-Native#5550] [Ideas]
If we pull the radio stations out of Streaming, then we have plenty of room to add the archived streams list, which would be nice. 

Then we'd have [ Upcoming | Archived | Webcams ] in Streaming, which I feel makes much more sense than if you have KSTO/KRLX in there too. 

I don't know what this would mean for the movie view; maybe that goes top-level too? ~~That would solve our question of how you see the list of past movies…~~

**Comment by drewvolz** [comment]
I’m not sure I want to show off the past movies

**Comment by hawkrives** [comment]
I've removed that from the proposal.
